crates/symbolicator/Cargo.toml

```diff
@@ -26,7 +26,8 @@ jsonwebtoken = "7.2.0"
 lazy_static = "1.4.0"
 lru = "0.7.0"
 num_cpus = "1.13.0"
-minidump = "0.9.4"
+minidump = "0.9.6"
+minidump-processor = "0.9.6"
```

Comment:
I would rather pull in the git version here, instead of via a `patch` directive.